docs/sql-programming-guide.md

```diff
@@ -964,7 +964,8 @@ Configuration of Parquet can be done using the `setConf` method on `SparkSession
     Sets the compression codec used when writing Parquet files. If either `compression` or
     `parquet.compression` is specified in the table-specific options/properties, the precedence would be
     `compression`, `parquet.compression`, `spark.sql.parquet.compression.codec`. Acceptable values include:
-    none, uncompressed, snappy, gzip, lzo, brotli, lz4, zstd.
+    none, uncompressed, snappy, gzip, lzo, brotli(need install brotliCodec), lz4, zstd(need install
+    ZStandardCodec before Hadoop 2.9.0).
```

Comment:
I would just add few lines for `brotli` and `zstd` below and leave the original text as is.